include/parsers/qasm3_parser/InstVisitor.hpp

```diff
@@ -0,0 +1,146 @@
+#pragma once
+
+#include <any>
+#include <iostream>
+#include <map>
+#include <sstream>
+#include <utility>
```

Comment:
```suggestion
#include <any>
#include <iostream>
#include <map>
#include <memory>
#include <sstream>
#include <utility>
```
You are missing that include in this file